templates/main.bicep

```diff
@@ -0,0 +1,358 @@
+targetScope = 'resourceGroup'
+
+// General parameters
+param location string = resourceGroup().location
+param tenantId string = subscription().tenantId
+param odhsiWebApiName string = 'ohdsi-webapi'
+param dockerRegistryServer string = 'https://index.docker.io/v1'
+param dockerImageName string = 'ohdsi/webapi'
+param dockerImageTag string = '2.12.1'
+param keyvaultName string ='keyvault-${uniqueString(resourceGroup().id)}'
+
+// Postgres parameters
+param serverEdition string = 'GeneralPurpose'
+param skuSizeGB int = 32
+param dbInstanceType string = 'Standard_D2s_v3'
+param availabilityZone string = '3'
+param version string = '14'
+param postgresDatabaseName string = 'atlas_db'
+param postgresSchemaName string = 'webapi'
+param flywayBaselineVersion string = '2.2.5.20180212152023'
+// Users and passwords
+param postgresAdminUsername string = 'postgres_admin'
+param postgresWebapiAdminUsername string = 'ohdsi_admin_user'
+param postgresWebapiAppUsername string = 'ohdsi_app_user'
```

Comment:
I think some of these at least should be just var as it's unlikely a user will need to override them.

Reply:
Done - all database related users are now vars